logging/google/cloud/logging/handlers/container_engine.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Logging handler for Google Container Engine (GKE).
+
+Formats log messages in a JSON format, so that Kubernetes clusters with the
+fluentd google cloud plugin installed can format their log messages so that
+metadata such as log level is properly captured.
+"""
+
+import logging.handlers
+
+from google.cloud.logging.handlers._helpers import format_stackdriver_json
+
+
+class ContainerEngineHandler(logging.StreamHandler):
+    """Handler to formats log messages in good fluentd GKE format.
```

Comment:
"to formats" --> "to format"?

Also "good fluentd FKE format"? What makes it good or bad? Don't you just mean the correct format?